static/app/components/group/tagFacets/tagFacetsDistributionMeter.tsx

```diff
@@ -116,41 +130,34 @@ function TagFacetsDistributionMeter({
             <div
               key={value.value}
               style={{width: pct + '%'}}
-              onMouseOver={() =>
+              onMouseOver={() => {
+                setHoveredValue(value);
                 _debounceTrackHover({
                   tag: title,
                   value: value.value,
                   platform: project.platform,
                   is_mobile: isMobilePlatform(project?.platform),
                   organization,
                   type: 'distributions',
-                })
-              }
+                });
+              }}
+              onMouseLeave={() => setHoveredValue(null)}
             >
-              <Tooltip
-                title={renderLegend(true)}
-                containerDisplayMode="block"
-                position="bottom"
-              >
-                {value.isOther ? (
-                  <OtherSegment
-                    aria-label={t('Other')}
-                    color={colors[colors.length - 1]}
-                  />
-                ) : (
-                  <Segment
-                    aria-label={t(
-                      'Add the %s %s segment tag to the search query',
-                      title,
-                      value.value
-                    )}
-                    color={colors[index]}
-                    {...segmentProps}
-                  >
-                    {index === 0 ? `${pctLabel}%` : null}
-                  </Segment>
-                )}
-              </Tooltip>
+              {value.isOther ? (
+                <OtherSegment aria-label={t('Other')} color={colors[colors.length - 1]} />
+              ) : (
+                <Segment
+                  aria-label={t(
+                    'Add the %s %s segment tag to the search query',
+                    title,
+                    value.value
+                  )}
+                  color={colors[index]}
+                  {...segmentProps}
+                >
+                  {index === 0 && pctLabel > 6 ? `${pctLabel}%` : null}
```

Comment:
Why the 6% threshold? Is it just because anything less than 6% doesn't really fit the distribution segment? Maybe pulling this into a variable/adding a comment might help

Reply:
yup, if the segment is 6% or less, the label doesn't fit in the segment so we don't display it in that case. Added a comment!